en/developer_guide/media_standalone_display.rst

```diff
@@ -0,0 +1,186 @@
+Medias display
+==============
+
+Context
+-------
```

Comment:
Maybe smth like
"The section introduction the orchestraMediaBundle, a standalone bundle targeting media management/distribution/services/other? in a Symfony project. It can be used outside of the full orchestra stack so you can benefits from its features in any Symfony application."